docs/conduit/WritingTxnsToFile.md

```diff
@@ -0,0 +1,191 @@
+## Writing Transactions to Files Using Conduit
```

Comment:
Do you think we should have a tutorials directory?

Reply:
Probably, I have a couple of other write ups that I'm working on that would be "tutorials" I guess.